templates/people/person_page.html

```diff
@@ -12,29 +12,48 @@
             {% if page.body %}
                 {% include_block page.body %}
             {% endif %}
+
+            {% if recent_projects %}
+                <div class="block-tile_block">
```

Comment:
this is an unusually formatted classname — something from the existing site, I assume?

Reply:
yeah, that's a wagtail/django magic automatic wrapper. there is actually a way to get those not to render, but i remembered that a bit late 😅